FixingLimits (#114560)
Azure/azure-content: articles/postgresql/flexible-server/concepts-limits.md

- We recommend setting alert rules for `storage used` or `storage percent` when they exceed certain thresholds so that you can proactively take action such as increasing the storage size. For example, you can set an alert if the storage percent exceeds 80% usage.
- If you're using logical replication, then you must drop the logical replication slot in the primary server if the corresponding subscriber no longer exists. Otherwise the WAL files start to get accumulated in the primary filling up the storage. If the storage threshold exceeds certain threshold and if the logical replication slot isn't in use (due to non-available subscriber), Flexible server automatically drops that unused logical replication slot. That action releases accumulated WAL files and avoids your server becoming unavailable due to storage getting filled situation. 

> - We don't support the creation of tablespaces, so if you're creating a database, don’t provide a tablespace name, PostgreSQL will use the default one that is inherited from the template database. It is unsafe to provide a tablespace like the temporary one since we cannot ensure that such objects will remain persistent after server restarts, HA failovers, etc.

Reviewer: ```suggestion
- We don't support the creation of tablespaces, so if you're creating a database, don’t provide a tablespace name. PostgreSQL will use the default one that is inherited from the template database. It's unsafe to provide a tablespace like the temporary one because we can't ensure that such objects will remain persistent after server restarts, HA failovers, etc.
```
